### ISSUE TITLE
Can't install v0.3.1 via homebrew

### ISSUE THREAD

**Issue by LubosAngus** [passbolt/go-passbolt-cli#45]
**Describe the bug:**
Can't install v0.3.1 via homebrew

**To Reproduce:**
Open terminal, run `brew install passbolt/tap/go-passbolt-cli`
```shell
==> Fetching passbolt/tap/go-passbolt-cli
==> Downloading https://github.com/passbolt/go-passbolt-cli/releases/download/v0.3.1/go-passbolt-cli_0.3.1_darwin_arm64.tar.gz
curl: (22) The requested URL returned error: 404

Error: go-passbolt-cli: Failed to download resource "go-passbolt-cli"
Download failed: https://github.com/passbolt/go-passbolt-cli/releases/download/v0.3.1/go-passbolt-cli_0.3.1_darwin_arm64.tar.gz
```

[https://app.warp.dev/block/pUqTupOZuFb8pgmOf0uT0m](https://app.warp.dev/block/pUqTupOZuFb8pgmOf0uT0m)

**Output when using --debug (you should censor this):**
-

**Passbolt Server Version  (please complete the following information):**
 - Edition: Community
 - Version 0.3.1

**go-passbolt-cli Version (please complete the following information):**
 - OS: macOS
 - Version 13.4

**Additional context**
I think it's more than clear.

**Comment by speatzle** [comment]
Hi, v0.3.1 has not yet been release due to a issue, it seems that homebrew uses the latest tag instead of the latest release which is causing this issue, for now you can tell homebrew to install v0.3.0 instead, which should work, ill see if i can release v0.3.1 Soon.

**Comment by speatzle** [comment]
Hi, i just released v0.3.1.
Can you check if you can install now?
Thanks

**Comment by LubosAngus** [comment]
Many thanks, it's working now.
